packages/template-retail-react-app/app/pages/account/wishlist/partials/wishlist-primary-action.test.js

```diff
@@ -97,3 +97,11 @@ test('the View Options button', async () => {
         {timeout: 5000}
     )
 }, 30000)
+
+test('bundle in wishlist renders the View Full Details button', async () => {
+    const productBundle = mockWishListDetails.data[4]
+    renderWithProviders(<MockedComponent variant={productBundle} />)
+
+    const link = await screen.findByRole('link', {name: /view full details/i})
```

Comment:
```suggestion
    const link = await screen.findByText(/view full details/i)
```
[nit] `byText` is more performant than `byRole` (even though `byRole` is the recommendation from the official docs). @vmarta looked into this if you care to learn more.